src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 0;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin-top: 1em;
+  margin-bottom: 1em;
```

Comment:
try to write margin in one line

Reply:
Done.
Should I create new pull request?
```sugestion
  margin: 1em auto;
```